.travis.yml

```diff
@@ -25,12 +25,16 @@ branches:
     - /^amp-release-.*$/
     - /^revert-.*$/
 addons:
+  apt:
+    packages:
+      - protobuf-compiler
+      - python-protobuf
```

Comment:
nit: Only the Validator Tests job needs these packages, but it takes ~5 seconds so not a big deal that all VMs install them

Reply:
Yeah, it was a trade off between maintaining separate install paths for different shards and doing it at once at the top.